Week2/todo_app/config.json

```diff
@@ -0,0 +1,5 @@
+{
+    "host": "127.0.0.1",
+    "username": "root",
+    "password": "hamid615243"
```

Comment:
You should use gitignore to prevent accidentally committing this, and you should also be checking your commits (with `git status` and `git diff --staged`) before you commit to make sure you're not about to commit something by accident.

Your problem was naming the file `.gitignore.json` -it's not a json file, the file should just be called `.gitignore`.